iam/snippets/src/main/java/CreateRole.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.google.api.gax.rpc.AlreadyExistsException;
+import com.google.cloud.iam.admin.v1.IAMClient;
+import com.google.iam.admin.v1.CreateRoleRequest;
+import com.google.iam.admin.v1.Role;
+import com.google.iam.admin.v1.Role.RoleLaunchStage;
+import java.io.IOException;
+import java.util.Arrays;
+
+// [START iam_create_role]
```

Comment:
Region tag must include the imports. Only exclude the package definition and the license header.

Reply:
This applies to all the samples

Reply:
@averikitsch fyi the existing samples aren't using a package. what should this package be?

https://github.com/GoogleCloudPlatform/java-docs-samples/tree/main/iam/snippets/src/main/java